### ISSUE TITLE
Trails Brandmark, Logotype, and Theming

### ISSUE THREAD

**Issue by yejodido** [trailsjs/trails#47]
Hi everyone, just wanted to share with you our current branding efforts courtesy @jonhargreaves .

The second photo is a quick mock of the logo applied to the TrailsMix app, wanted to show a quick sense of it "in action" 🌲

![spreadsheet](https://cloud.githubusercontent.com/assets/5909827/12060373/09b79266-af3c-11e5-8334-7d087375d971.png)

<img width="1295" alt="trailsmixnewlogo" src="https://cloud.githubusercontent.com/assets/5909827/12060380/249cff12-af3c-11e5-9c96-ae7ac488962b.png">

**Comment by mikehostetler** [comment]
I like it a lot. 

Few nitpicks:

* The sentence should read, "Blaze your own Trail" (drop the S)
* I prefer the darker background (Kilamanjaro) with the lighter logo.  The contrast says "Forest" to me, which is where Trails typically lead.

I'd love to see a full style guide played out a bit more, so there's consistent fonts and such.  Not sure if @jonhargreaves does any illustration work, but more illustrations with a campfire or something would be cool too.

What about a extended logo for "Trailpacks" as well?

**Comment by yejodido** [comment]
@mikehostetler 

Yea I can change that, was trying to be clever with the added 's', similar to "TrailsMix":

<img width="943" alt="screen shot 2015-12-31 at 10 44 00 am" src="https://cloud.githubusercontent.com/assets/5909827/12065777/6f4639ba-afab-11e5-8a11-cce73e309ece.png">

**Comment by mikehostetler** [comment]
Ahh, I see that now! I like that, it almost keeps the "Trails" as the anchor and allows us to build brand extensions off of it.

**Comment by metzuda** [comment]
+1 Kilimanjaro!

**Comment by jaumard** [comment]
My favorite is the one use for TrailsMix :) (love the colors) so cello/light forest. Doesn't need a background in my opinion ^^, it make logo heavier

**Comment by troy0820** [comment]
+1 with what @jaumard said.  I like tent :+1:

**Comment by jaumard** [comment]
This PR is related guys :D https://github.com/trailsjs/trailpack-core/pull/3 ^^

**Comment by yejodido** [comment]
Hi everyone, it's come to our attention that our logo is very similar to the logo at http://tent.io

We're actively working on a new logo (the tent icon) but of course we'll keep the pallette and logotype to ensure that the new voice speaks loudly, clearly, and coherently towards our goal with Trails.

Be on the lookout for the new designs coming soon :)

**Comment by kokujin** [comment]
I think the "tent" is overkill anyway, what does it have to to with trail-blazing? The "trails" text logo would suffice, a little simplified perhaps as it is with good logos

**Comment by tjwebb** [comment]
@kokujin we do need an "icon" though, like for Twitter and so forth.

**Comment by kokujin** [comment]
agreed @tjwebb, may I then suggest that the icon should show an abstraction of a trail? Look at the "Grandad" of these frameworks, Rails a.k.a  Ruby on rails. The icon logo has a stretch of rail tracks in it.

**Comment by tjwebb** [comment]
I sort of like the idea of turning the github fork icon into a trail of some sort. @yejodido @jonhargreaves how are things going on the update?

**Comment by yejodido** [comment]
@tjwebb sort of like...
![12467890_10208499065103892_1455661337_n](https://cloud.githubusercontent.com/assets/5909827/12126765/21bf6a9e-b3c1-11e5-8a31-135f1b5c4f1b.jpg)

Also for consideration
![12498558_10208498973581604_1170925309_n](https://cloud.githubusercontent.com/assets/5909827/12126768/24fa46fc-b3c1-11e5-8619-4d277d1148ec.jpg)
![12498722_10208498973701607_685389846_n](https://cloud.githubusercontent.com/assets/5909827/12126773/2cf76984-b3c1-11e5-88b9-9b6ef79b588d.jpg)
![12511801_10208498973621605_1588572557_n](https://cloud.githubusercontent.com/assets/5909827/12126777/306088f8-b3c1-11e5-8272-489135b4ec46.jpg)

**Comment by tjwebb** [comment]
first one: "solidify" leaves, remove some of the lines
second: make it look more like a real leaf
3rd: like a lot, try some variations. try using golden ratio for the rectangles

**Comment by jaumard** [comment]
@yejodido nice work :) I prefer the ones with 2 and 3 leaves. For the one with 2 leaves make the left leaf above the other one, cause the "fork" will surpass the original project ;) 

PS : But the tent looked better lol

**Comment by troy0820** [comment]
Question concerning the fork one.  Do we want to do that?  Big `issue` about this being an unintentional fork of sails (I know it's not but you know).  Didn't want a logo to have that distinction.  The leaves are great IMO but we don't want to copy a company like mint but an evergreen leaf is something cool.  I like the rectangles.  The tent was cool :disappointed: . We can always make a path somewhere with a whole bunch of server options.  Like a cluster of options with a guy exploring the trail or something.

**Comment by jaumard** [comment]
:+1: yeah @troy0820 about fork :/ a lot of people will make the shortcut easily.
Why not a comet trail ? The nail can make the T bar and the comet the i point...

I try something more traditional about leaf also : 
![trailslogo](https://cloud.githubusercontent.com/assets/2970449/12128463/bfcb55b8-b3fd-11e5-97d1-36ab3afeb812.png)

**Comment by codingcampbell** [comment]
Disclaimer: not a graphic designer by a long shot, but throwing in my .02c anyway.

I like the leaf direction better than the tent, but it also seems kinda generic and not directly related to trails. I feel like there's a ton of leaf logos out there, maybe that's just because of where I work.

The tent itself would better if you dropped the side of it to square it off (good for iconification too):

![untitled-2](https://cloud.githubusercontent.com/assets/311270/12131456/ca49d130-b3e0-11e5-8e3e-4b44c828d437.png)

Regardless of direction you choose for the symbol portion of the logo.. I think you should leave the typography WordArt behind. IMO it comes off as too try-hard like it _really_ wants me to notice the "nature" of the theme.

Good luck!

**Comment by codingcampbell** [comment]
Oops, nevermind, it looks like https://tent.io/ uses exactly that logo anyways :sweat_smile:

**Comment by tjwebb** [comment]
@codingcampbell I agree, actually, but this is the problem with that: https://tent.io :) After we ended up on hackernews, the creators of tent.io reached out to us and kindly let us know that we were stepping on their logo. This is why we're moving away from the tent idea.

**Comment by codingcampbell** [comment]
@tjwebb yeah, stanley kindly pointed that out to me _after_ I shopped the thing >_<

One more thing I'll spam the thread with to hopefully inspire the real designers in this thread: Beyond leaves, there are quite a few "rugged" aesthetics people naturally attribute to camping. Perhaps a but generic, but some quick wins would be like snowcapped mountains, bears, pine trees, etc. They are also very silhouette friendly. Some examples:

![untitled-1](https://cloud.githubusercontent.com/assets/311270/12131709/20240100-b3e3-11e5-9eab-9b0db260d773.jpg)


'rugged' look:

![untitled-2](https://cloud.githubusercontent.com/assets/311270/12131737/329e0272-b3e3-11e5-9c1f-fd735e5d102a.jpg)

You could even adopt an animal soulhouette potentially:

![untitled-3](https://cloud.githubusercontent.com/assets/311270/12131757/5f7b43ea-b3e3-11e5-8db0-218711f674ef.jpg)

That's all I have to add. I hope the rest of your logo search is quick and painless.

**Comment by metzuda** [comment]
What if, instead of the tent, we keep the same triangular shape but have a sort of winding trail going up towards the top? Imagine a single, lonely mountain with a path to the peak. 

Like this:

![a trail](https://cloud.githubusercontent.com/assets/8866130/12138925/84cf593a-b454-11e5-80cf-8cedbcd6520a.jpg)

Except not drawn by a eight year-old.

**Comment by Eein** [comment]
Just some input: It might be a good idea to keep the text on the logo as readable as possible and represent with a mark, this way it lends well with marketing future endeavors/cons/etc. The proposed trails text is too busy wouldn't lend well to paper/marketing materials when/if that time comes.

**Comment by tjwebb** [comment]
> Except not drawn by a eight year-old.

LOL :+1: 

> What if, instead of the tent, we keep the same triangular shape but have a sort of winding trail going up towards the top?

Hey @jonhargreaves would you mind prototyping this out as well? Just make sure it doesn't too closely evoke the [Eye of Providence](https://en.wikipedia.org/wiki/Eye_of_Providence)

**Comment by tjwebb** [comment]
@codingcampbell I like the stock mountains you sent over. We're working on some background drawings (e.g. for presentation slides and so forth) that might benefit from those ideas.

**Comment by jaumard** [comment]
It can be cool to have a logo for Trailpacks also :) with same type ect...

**Comment by nrempel** [comment]
+1 for @metzuda's suggestion

**Comment by jonhargreaves** [comment]
@tjwebb Lol! Yes, I will get you the prototype without the Eye of Providence as soon as possible.

**Comment by jaumard** [comment]
Any updates for the new logo ? :D

**Comment by yejodido** [comment]
Hey everyone! Thanks for the patience and suggestions.

**pulls away curtain**

![trailslogo](https://cloud.githubusercontent.com/assets/5909827/12692785/3324c7c2-c6ca-11e5-8b05-9e4e044e4d42.png)

_Ta-da!_ 

At this point we're actively working on the website and documentation, as well as finishing up TrailsMix for an early beta launch.
